src/components/filesystem/index.js

```diff
@@ -1514,7 +1514,43 @@ module.exports = {
             }.bind(this));
             e.preventDefault();
         },
-
+        createTextFile: function() {
+            this.closeMenu();
+            this.prompt_placeholder='File name';
+            this.prompt_message='Enter a file name';
+            this.prompt_value='';
```

Comment:
maybe default to example.txt?

Reply:
But then the prompt_message would not be displayed.
![prompt](https://user-images.githubusercontent.com/5311499/91684264-de6ec800-eb4e-11ea-80d1-34d95704a638.png)